internal/auth0/connection/resource_scim.go

```diff
@@ -0,0 +1,88 @@
+package connection
+
+import (
+	"context"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/auth0/terraform-provider-auth0/internal/config"
+	internalError "github.com/auth0/terraform-provider-auth0/internal/error"
+	internalSchema "github.com/auth0/terraform-provider-auth0/internal/schema"
+)
+
+// NewSCIMConfigurationResource will return a new auth0_scim_configuration (1:1) resource.
+func NewSCIMConfigurationResource() *schema.Resource {
+	return &schema.Resource{
+		Schema:        scimResourceSchema,
+		CreateContext: createSCIMConfiguration,
+		UpdateContext: updateSCIMConfiguration,
+		ReadContext:   readSCIMConfiguration,
+		DeleteContext: deleteSCIMConfiguration,
+		Importer: &schema.ResourceImporter{
+			StateContext: internalSchema.ImportResourceGroupID("client_id"),
```

Comment:
```suggestion
			StateContext: internalSchema.ImportResourceGroupID("client_id"),
```

This seems wrong, shouldn't it just be a passthrough and using the connection ID?

Reply:
yes, that one looks like it must have been a cut/paste error